include/zephyr/arch/arm/mpu/arm_mpu_v8.h

```diff
@@ -309,6 +295,13 @@
 	}
 #endif /* CONFIG_MPU_ALLOW_FLASH_WRITE */
 
+#define REGION_DEVICE_ATTR(base, size)                                                             \
+	{                                                                                          \
+		/* AP, XN, SH */                                                                   \
+		.rbar = NOT_EXEC | P_RW_U_NA_Msk | NON_SHAREABLE_Msk, /* Cache-ability */          \
+		.mair_idx = MPU_MAIR_INDEX_DEVICE,                    /* Region Limit */           \
```

Comment:
remove or change "Region Limit" to be "MAIR Index"?

Reply:
Thanks. Removed.